scripts/setup-devnet.sh

```diff
@@ -1,35 +1,52 @@
 #!/bin/bash
 
+# useful dir relative to this script
 MONERO_DIR="../monero-x86_64-linux-gnu-v0.17.3.0"
+LOG_DIR="../log"
```

Comment:
could you make this `$TMPDIR/swapd_log` or something like that? so it automatically goes into the temp directory?

Reply:
found an elegant way with TMPDIR and a default `/tmp` in case nothing's set